internal/call/option.go

```diff
@@ -5,5 +5,13 @@ package call
 // Option defines the call Info adjustment function
 type Option func(c *Info)
 
+// Options contains options for all tracing calls.
+type Options struct {
+	// DisableInfoLogging determines if info logging should be disabled or not
+	// when a call is finished. This is useful for calls that are expected to
+	// be very frequent, such as HTTP requests.
```

Comment:
Be explicit with booleans:
```suggestion
	// DisableInfoLogging turns off per-call info logging if set to true. If false, every successful
	// (statuscodes.CategoryOK) call will have an Info line emitted.
```